hw3/01-routing.js

```diff
@@ -1,31 +1,28 @@
-const http = require('http');
-const port = process.env.PORT || 5000;
+const http = require(`http`);
+const port = process.env.PORT || 5001;
 
```

Comment:
Hey @Shweta-Tapele, This exercise needs to be completed using Express and I don't see any imports or any usage of ExpressJS in this file. It looks like the same file from the previous exercise. Can you look into this?